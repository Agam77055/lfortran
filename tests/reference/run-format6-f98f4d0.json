{
    "basename": "run-format6-f98f4d0",
    "cmd": "lfortran --no-color {infile}",
    "infile": "tests/format6.f90",
    "infile_hash": "ea6867f31214df92d0d0d39eeab15571a198af2b462c18cc925f560c",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "run-format6-f98f4d0.stdout",
    "stdout_hash": "03404382de7b6c481ccfd5786295bf4c251b6254fdc2cc5c20da7b00",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}