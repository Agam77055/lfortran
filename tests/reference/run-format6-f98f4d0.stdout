 +12.1+122.1 122.1
 +011
+011
+11
  +11
-11
-1111
***
11
        +0.1234567871D+05
+0.123456789101111989708674472293960984310250256775136653828155E-61
+12.3457E+03   +12.8E+03
 +1.0
